tap_trello/streams.py

```diff
@@ -220,30 +220,38 @@ def get_records(self, format_values, additional_params=None):
             )
 
         for rec in records:
-            yield self.modify_record(rec, parent_id_list = format_values, custom_fields_map = custom_fields_map)
+            yield self.modify_record(rec, parent_id_list = format_values, custom_fields_map = custom_fields_map, dropdown_fields_map = dropdown_fields_map)
 
 
     def sync(self):
         for rec in self.get_records(self.get_format_values()):
             yield rec
 
 class AddCustomFields(Mixin):
-    def build_custom_fields_map(self, **kwargs):
+    def build_custom_fields_maps(self, **kwargs):
         custom_fields_map = {}
+        dropdown_options_map = {}
```

Comment:
This should probably be called `dropdown_fields_map` for consistency.

Reply:
Actually, `dropdown_options_map` is probably better across the tap, since that's more in line with what it is